packages/exhibitor/build

```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
 
+set -x
```

Comment:
Was this for debugging, or do we want to see shell output?

Reply:
> Was this for debugging, or do we want to see shell output?

I am of the opinion that we generally want to see that output, and various other `build` files of other pkgpanda packages set this shell option.